doc/releases/dumpling.rst

```diff
@@ -721,7 +721,7 @@ Notable changes since v0.61 "Cuttlefish"
 * osd: fix for an op ordering bug
 * osd: fix handling for split after upgrade from bobtail
 * osd: fix incorrect mark-down of osds
-* osd: fix internal heartbeart timeouts when scrubbing very large objects
+* osd: fix internal heartbeat timeouts when scrubbing very large objects
```

Comment:
Not to be confused with Neil Peart beats

Reply:
I echo @zdover23 's thoughts about not changing the old release notes, at least not the commit messages.  That means `bobtail` through at least `mimic`, above and below this comment.